src/components/Header/Header.jsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from "react";
 import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import icon_close from "../../assets/icon-close.svg";
+import logo from "../../assets/logo.svg";
+import styles from "./Header.module.css";
 
 const Header = () => {
+  const [mobileMenu, setMobileMenu] = useState(false);
+  const toggleMenu = () => {
+    mobileMenu ? setMobileMenu(false) :
+   setMobileMenu(true);
+  };
   return (
+  
     <header className={styles.header}>
-      <nav>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/contact">Contact Us</Link></li>
+      <img src={logo} alt="myteam logo" className={styles.logo} />
+      <nav className={styles.header-nav}>
```

Comment:
CSS Modules don't support calling hyphenated (aka "kebab-case") class names like `styles.header-nav`. If you use hyphenated class names, use the form `styles['header-nav']` to call the styles. 

OR you can use camelCase class names instead: `styles.headerNav` will work great. This tends to be how React devs do it, to the best of my knowledge.

Reply:
Apologies for suggesting kebab-case-names when that's not the most common in React. I saw that you already had one in the file, so I just went with that rather than trying to rename that immediately, and I would do it differently next time.